Stable branch: Add OpenCL headers

user1-github opened the pull request:
Needed to build Wine with OpenCL support

user1-github committed:
Stable branch: Add OpenCL headers

user1-github commented:
That's weird. Why is strip-components suddenly an unknown property?

Erick555 commented:
it may be related to https://github.com/flathub-infra/flatpak-builder-lint/commit/bdfae494f50586e32f55331e4b8c59856d38cdf6

user1-github commented:
So is this a new bug due to this commit?
It is supposed to abort on unknown properties, but strip-components is a documented property according to [this Flatpak documentation](https://docs.flatpak.org/en/latest/module-sources.html#archive-sources)

Erick555 commented:
you may open issue in that repo

user1-github commented:
Ok, I'll first trigger another build to make sure it wasn't just a glitch

user1-github commented:
bot, build

user1-github committed:
Remove strip-components from 7zip git source

Turns out the manifest validation failure was due to strip-components being set on a git source.
